python_modules/libraries/dagster-dbt/dagster_dbt/asset_decorator.py

```diff
@@ -206,6 +206,53 @@ def my_dbt_assets(context: OpExecutionContext, dbt: DbtCliResource, config: MyDb
                     dbt_build_args += ["--full-refresh"]
 
                 yield from dbt.cli(dbt_build_args, context=context).stream()
+
+        Creating partitions for dbt assets:
+
+        :any:`dbt_assets` produces multiple assets, but the :any:`asset_partitions_time_window_for_output`
+        method expects a single output. Since all dbt assets have the same
+        partition definition, we can provide the first dbt asset to the method
+        in order to retrieve the first and last partition window.
+
+        We pass these as variables to the dbt CLI which can be read as jinja
+        macro variables.
```

Comment:
Seems more like `reference.mdx` material

Reply:
Makes sense, wasn't sure how much detail to provide here, will move it over.